packages/server/modules/workspaces/services/management.ts

```diff
@@ -19,8 +19,8 @@ import { MaybeNullOrUndefined, Roles } from '@speckle/shared'
 import cryptoRandomString from 'crypto-random-string'
 import {
   deleteStream,
-  grantStreamPermissions as repoGrantStreamPermissions,
-  revokeStreamPermissions as repoRevokeStreamPermissions
+  grantStreamPermissionsFactory,
```

Comment:
we should not invite repositories into services directly, this should be a domain / operation interface type import